lib/axp209.js

```diff
@@ -85,11 +95,38 @@ AXP209.prototype._readButton = function() {
   return (value ? 1 : 0);
 };
 
+AXP209.prototype._readVcc18Adc = function() {
+  // enable 12-bit ADC for VCC 1.8 
+  var p = this._i2c.readRegister(0x85, 1)[0]; // 0-2v range or .7 - 2.7v range?
+  var rhigh = this._i2c.readRegister(0x64, 1)[0].toString(16);
+  var rlow = this._i2c.readRegister(0x65, 1)[0].toString(16);
+  var r = parseInt(rhigh + rlow, 16);
```

Comment:
so I realized, my initial code here was correct, but if there's a more elegant way to do it I'm open to change it.

Basically there should be no leading 0 added to that toString calculation.   `voltage` returns a value between 0 and ~19995  which is correct in millivolts because thats between 0 and 1.99 V.     If you set register `0x85` to `0x01` in the configure (instead of `0x01`),  the range returns between 7000 and 27000, which corresponds to 0.7V and 2.7V    This aligns with the documentation on the CHIP thread.